reference/configuration/doctrine.rst

```diff
@@ -485,4 +485,137 @@ can be placed directly under ``doctrine.orm`` config level.
 This shortened version is commonly used in other documentation sections.
 Keep in mind that you can't use both syntaxes at the same time.
 
+Custom Mapping
+--------------
+
+It's possible to not rely on the auto mapping and set all the configuration
+on your own.
+
+For example, you can overwrite the configuration directory for a given bundle.
+Keep in mind that the bundle you want to overwrite has to exist.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+    
+        doctrine:
+            # ...
+            orm:
+                # ...
+                auto_mapping: true
+                mappings:
+                    # ...
+                    AppBundle:
+                        dir: Resources/config/doctrine
+                        
+    .. code-block:: xml
+    
+        <?xml version="1.0" charset="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:doctrine="http://symfony.com/schema/dic/doctrine">
+        
+            <doctrine:config>
+                <doctrine:orm auto-mapping="true">
+                    <mapping name="AppBundle" dir="Resources/config/doctrine" />
+                </doctrine:orm>
+            </doctrine:config>
+        </container>
+        
+    .. code-block:: php
+    
+        $container->loadFromExtension('doctrine', array(
+            'orm' => array(
+                'auto_mapping' => true,
+                'mappings' => array(
+                    'AppBundle' => array('dir' => 'Resources/config/doctrine'),
+                ),
+            ),
+        ));
+
+
+You can also create new mappings, for example outside of the Symfony folder.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+    
+        doctrine:
+                # ...
+                orm:
+                    # ...
+                    mappings:
+                        # ...
+                        SomeEntity:
```

Comment:
I think that's okay too. It just needs to be consistent throughout the different formats.